server/lib/github-client.js

```diff
@@ -1,12 +1,20 @@
-const host = 'http://cyf-github-api.herokuapp.com';
 const fetch = require('isomorphic-fetch');
+const gitApi = 'https://api.github.com';
 
-function fetchFromGithub (path) {
-	// write something that will fetch json information from github
+fetchFromGithub = path => {
+  return (
+    fetch(`${gitApi}${path}`)
+    .then(apiRes => apiRes.json())
+    .then(json => json)
+    .catch(err => console.error(err))
+    );
 }
 
-module.exports.getUserProfile = function (username) {
-	return fetchFromGithub(`/users/${username}`);
-};
+module.exports.getUserProfile = username => fetchFromGithub(`/users/${username}`);
 
-// add a method to retrieve user pull requests here
+module.exports.getUserPullRequests = username => {
+  return (
+    fetchFromGithub(`/users/${username}/events/public`)
+    .then(data => data.filter(value => value.type === 'PullRequestEvent'))
```

Comment:
the `.then` should be indented a level. 

So, this `data => data.filter(value => value.type === 'PullRequestEvent')` isn't hugely easy to understand at a glance for someone who doesn't know what your code is, sometimes  it's nice to make things more readable and abstract some of the logic away. Do you think you could write a function to do the filtering so that this could be changed to something a bit easier to read like...

```js
fetchFromGithub(`/users/${username}/events/public`)
  .then(filterByEventType('PullRequestEvent'))
```

If you can do this it'd be great as in the future we could easily filter by different event types like Issues and Comments just by calling your new function with a different event type